src/librustc/lint/builtin.rs

```diff
@@ -279,6 +279,12 @@ declare_lint! {
     "detects name collision with an existing but unstable method"
 }
 
+declare_lint! {
+    pub DUPLICATE_ASSOCIATED_TYPE_BINDING,
```

Comment:
Could you add this lint to the list of deprecation lints?
(See e.g. `PARENTHESIZED_PARAMS_IN_TYPES_AND_MODULES` as an example.)

Reply:
Also `duplicate_associated_type_binding` -> `duplicate_associated_type_bindings` (see https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints).

Reply:
Fixed.